articles/connections/social/google.md

```diff
@@ -10,56 +10,47 @@ alias:
  - google-oauth2
 seo_alias: google
 ---
-# Connect your app to Google
-
-To connect your Auth0 client to Google, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
+# Connect Your App to Google
 
 ::: note
-This doc refers to the client steps to connect your client to Google. If you are looking to manage authentication in your app, see [Next Steps](#next-steps) below.
+This doc only shows you how to connect your client to Google. To manage your app's authentication, see [Next Steps](#next-steps) for links to the appropriate docs.
```

Comment:
I think we should remove this note, wdyt? We can't have notes just because someone might not scroll down or use the article dropdown menu.